include/LLVMSPIRVLib.h

```diff
@@ -137,6 +137,18 @@ using SpecConstInfoTy = std::pair<uint32_t, uint32_t>;
 bool getSpecConstInfo(std::istream &IS,
                       std::vector<SpecConstInfoTy> &SpecConstInfo);
 
+struct SPIRVHeaderData {
+  std::uint32_t MemoryModel = 0x7fffffff; // MemoryModelMax
+  std::uint32_t AddressingModel = 0x7fffffff;
+  SPIRV::VersionNumber Version;
+  std::vector<std::uint32_t> Capabilities;
```

Comment:
SPIRVWord type may be used for uint_32 values and SPIRVWORD_MAX as default value.